tendrl/commons/flows/exceptions.py

```diff
@@ -1,9 +1,8 @@
+
+
 class FlowNotImplementedError(NotImplementedError):
-    def __init___(self, err):
-        self.message = "run function not implemented. %s".format(err)
+    pass
 
 
 class FlowExecutionFailedError(Exception):
-    def __init___(self, err):
-        self.message = "Flow Execution failed. Error:" + \
-                       " %s".format(err)
+    pass
```

Comment:
Why are these 2 exceptions changed?

@anmolsachan @nathan-weinberg

Reply:
I think @nathan-weinberg changed them because : 
1) The code wasn't being used anywhere. Removing it gave a better coverage.
2) This kind msg definition is being deprecated. 
3) Giving a definition for init is also not required as we can directly pass the message in the exception object. If we need the strings ''' "run function not implemented. %s ''' and ''' "Flow Execution failed. Error:" ''' to be here, we can keep them.

Reply:
@anmolsachan 

1) The exception class "FlowExecutionFailedError" is used in a lot of places eg: https://github.com/Tendrl/commons/blob/master/tendrl/commons/flows/configure_monitoring/__init__.py#L14

2) Is there a link to the issue or topic about deprecation of such messages, or could you create an issue

Reply:
Also here is the [link](https://stackoverflow.com/questions/1272138/baseexception-message-deprecated-in-python-2-6) regarding deprecation of such messages; the exceptions should still work as expected, I changed them for the same reasons Anmol mentioned.

Reply:
@r0h4n My bad, I just saw FlowNotImplementedError. FlowExecutionFailedError is used at a lot of places. @nathan-weinberg , I think we should revert the changes for now, otherwise it will require a lot of changes in code. As far as deprecation is concerned, it is a issue, we will make a new new issue and then propose the changes.